packages/design-system/src/components/commonBtn/CommonBtn.tsx

```diff
@@ -0,0 +1,45 @@
+import { cva } from 'class-variance-authority';
+interface CommonBtnProps {
```

Comment:
만약 button에서 사용하는 기본 속성도 props로 받게 하고 싶다면 아래와 같이 기존 타입을 extends해서 사용할 수 있어요!

```suggestion
interface CommonBtnProps extends ButtonHTMLAttributes<HTMLButtonElement> {
  // props 관련 type 정의
}
```

이렇게 하면 button 태그에 `{...props}`를 작성해서 기존 button 속성을 네이밍 그대로 사용해서 props 로 넘기면 적용돼요! 재사용하는 컴포넌트에 많이 사용합니다 👍